ingest/src/pipeline/storage.py

```
@@ -0,0 +1,212 @@
+                f"(got {len(nodes)} nodes, {len(embeddings)} embeddings)"
+            )
+
+        if not nodes:
+            logger.warning("No vectors to upsert")
+            return 0
+
+        logger.info(f"Upserting {len(nodes)} vectors to Qdrant")
+
+        # ポイントの作成
+        points = []
+        for node, embedding in zip(nodes, embeddings):
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)